Configuration/TypoScript/Setup/Part1PageviewSection.typoscript

```diff
@@ -0,0 +1,8 @@
+// Part 1: PAGEVIEW section
+page.10 = PAGEVIEW
+page.10 {
+  paths {
+    0 = EXT:site_package/Resources/Private/Templates/
+    10 = {$page.template.path}
```

Comment:
```suggestion
    
```

Reply:
Why would you delete that?

Reply:
This is considered as a common best practice. This way the path can be easily overriden just by setting a constant. Many extensions and sitepackages do this by default